harvester/sources/models/edurep.py

```diff
@@ -1,12 +1,22 @@
+import logging
```

Comment:
This file is also being edited, while it first should have been moved. In this case it not only confused GIT, but even me :smiley: